src/index.js

```diff
@@ -121,9 +121,9 @@ class Libp2p extends EventEmitter {
       })
     }
 
-    // enable/disable pubsub
-    if (this._config.EXPERIMENTAL.pubsub) {
-      this.pubsub = pubsub(this)
+    // start pubsub
+    if (this._config.pubsub.enabled) {
```

Comment:
@vasco-santos what do you think about changing this logic a bit. I think if `this._modules.pubsub` is set, and `this._config.pubsub.enabled !== false` we should use pubsub. This would avoid people needing to add **and** enable it. We do this for discovery. If you've configured it, there's a **really** good chance you want to use it :)

Reply:
I agree, this is a better dev ex